### PR TITLE
Turkish update

### PR DESCRIPTION
**Pull request by OzancanKaratas**
@Oreonan

### CODE REVIEW

File: Client/qtTeamTalk/languages/Turkish.ts [hunk 1]
```diff
@@ -1,6 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!DOCTYPE TS>
-<TS version="2.1" language="tr">
+<?xml version="1.0" ?><!DOCTYPE TS><TS language="tr" version="2.1">
```
Comment (Oreonan) [inline Client/qtTeamTalk/languages/Turkish.ts:1]: Why this change? QT translation tool will be replace this one line by 3 lines at next update I think.

Comment (OzancanKaratas) [reply Client/qtTeamTalk/languages/Turkish.ts:1]: No problem. It will work correctly. You can merge.
